client/verta/verta/registry/entities/_model.py

```diff
@@ -144,10 +169,22 @@ def set_version(self, *args, **kwargs):
         """
         Alias for :meth:`RegisteredModel.get_or_create_version()`.
 
-        """
+        """                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
```

Comment:
This line seems to have 1200 extra spaces 😅 

```suggestion
        """
```